src/airflow/airqo_etl_utils/data_validator.py

```diff
@@ -145,6 +145,20 @@ def process_for_big_query(
         dataframe = DataValidationUtils.remove_outliers(dataframe)
         return dataframe[columns]
 
+    @staticmethod
+    def process_for_message_broker_v2(data: pd.DataFrame) -> pd.DataFrame:
+        data["timestamp"] = pd.to_datetime(data["timestamp"])
+        data["timestamp"] = data["timestamp"].apply(date_to_str)
+        data.rename(
+            columns={
+                "mongo_id": "device_id",
+                "device_id": "device_name",
+                "tenant": "network",
```

Comment:
Perhaps this is where my worries are addressed from. Thanks @NoahNsimbe 

More like renaming network to tenant. 😀